third_party/DElight/DElightManagerC.cpp

```diff
@@ -212,7 +212,7 @@ catch(string sThrownMsg) {
         // No Error message file open, open one and write the thrown message to it
         ofdmpfile.open("eplusout.delightdfdmp");
         char msg[210];
-        sprintf(msg, "%s\n", sThrownMsg.c_str());
+        snprintf(msg, 210, "%s\n", sThrownMsg.c_str());
```

Comment:
sprintf needed to be converted to snprintf, which required adding an argument for the string size.  Runs happily.